src/credentials/FabricTable.h

```diff
@@ -404,6 +409,10 @@ class DLL_EXPORT FabricTable
      */
     CHIP_ERROR UpdateFabric(FabricIndex fabricIndex, FabricInfo & fabricInfo);
 
+#if CONFIG_IM_BUILD_FOR_UNIT_TEST
+    void SendUpdateFabricNotificationForTest(FabricIndex fabricIndex) { SendFabricUpdateNOC(fabricIndex); }
+#endif // CONFIG_IM_BUILD_FOR_UNIT_TEST
```

Comment:
I will soon have APIs that will allow us to trigger a NOC update directly, so this is only OK for now.